cli/tests/integration/lsp_tests.rs

```diff
@@ -4788,7 +4788,7 @@ fn lsp_completions_auto_import() {
         "source": "./b.ts",
         "data": {
           "exportName": "foo",
-          "exportMapKey": "foo|6799|file:///a/b",
+          "exportMapKey": "foo|6802|file:///a/b",
```

Comment:
We should find a way to remove this from the test somehow lol

Reply:
Yeah, it's been a PITA to update it every time :D